packages/home-spa/README.md

```diff
@@ -0,0 +1,47 @@
+# Welcome to home-spa 👋
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+
+> Home SPA for One Platform
+
+### 🏠 [Homepage](https://github.com/1-Platform/one-platform#readme)
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+### Development server
+```sh
+npm run dev
+```
+
+### Build
+```sh
+npm run build
+```
+
+## Run tests
+
+```sh
+npm run build
+npm t
+```
+
+## Color palette
+```
+https://codepen.io/hybridx18/full/rNabajv
```

Comment:
Providing personal CodePen account links are not always a good idea. I would highly recommend consolidating all project resources within the project itself. Otherwise, if you must you an external link, maybe you can create a gist under the 1-platform org, or a one-platform CodePen account.

Reply:
Understood